manifests/init.pp

```diff
@@ -231,7 +231,7 @@
       path    => "${confdir}/puppet.conf",
       section => 'main',
       setting => 'hiera_config',
-      value   => $hiera_yaml,
+      value   => '$confdir/hiera.yaml',
```

Comment:
Yes, but it is the default value and as far as I know this module does not support any config per environment.